test/e2e/suite-ingress/suite-ingress-resource/stream.go

```diff
@@ -130,11 +128,6 @@ spec:
       servicePort: 53
 `)
 			assert.Nil(ginkgo.GinkgoT(), s.CreateVersionedApisixResource(apisixRoute))
-			defer func() {
-				err := s.RemoveResourceByString(apisixRoute)
-				assert.Nil(ginkgo.GinkgoT(), err)
-			}()
```

Comment:
We should keep this, right?

This resource can't remove automatically

Reply:
The deletion of resources is based on the namespace. Why is this resource not in the scope?

Reply:
I remember wrong. :face_exhaling: